scheduler/config-k8s.edn

```diff
@@ -16,14 +16,21 @@
          ; because the minikube users and the local users may have different IDs, tests that require file permissions
          ; (like test_default_container_volumes) will fail without this set.
          :run-as-user "root"}
+ :autoscaler {:interval-seconds 180}
  :compute-clusters [{:factory-fn cook.kubernetes.compute-cluster/factory-fn
                      :config {:compute-cluster-name "gke-1"
                               ;; Location of the kubernetes config file. Hardcoded to the location specified by bin/make-gke-test-cluster
-                              :config-file "../scheduler/.cook_kubeconfig_1"}}
+                              :config-file "../scheduler/.cook_kubeconfig_1"
+                              :synthetic-pods {:image "gcr.io/google-containers/alpine-with-bash:1.0"
+                                               :max-pods-outstanding 128
```

Comment:
In addition to max outstanding pods, should we limit the outstanding cpu/mem so that we do not scale up to too many nodes (GKEs tendency to 'spread' the load across new nodes doesn't help).

Reply:
@rmanyari Thoughts on this?

Reply:
I think number of pods is fine for now. For a v2, I'd consider breaking down this value by pool. Essentially, you want a higher value for pools that grow large (say spot) and smaller for others, to avoid too much over-provisioning.